API/controllers/DjangoMlController.mjs

```diff
@@ -0,0 +1,48 @@
+import { ExpressValidator } from "express-validator";
+import "express-validator"
+import axios from "axios";
+
+const { validationResult } = new ExpressValidator
+
+/*
+    returns different traditional and zscore data for both teams given
+    and mean and std for league in general.
+*/
+export const findLRPred = async (req, res, next) => {
+    try {
+        const errors = validationResult(req);
+
+        if (!errors.isEmpty()) {
+            return res.status(400).json({
+                success: false,
+                errors: errors.array(),
+            })
+        }
+
+        const params = {
```

Comment:
the parameters entered here should be the team names. Then a search should be done in the MongoDB for the respective stats, and those stats would be sent to the ml service

Reply:
I wanted it so that it could editable in the frontend, if someone says for the game they predict the team will score 4 more rebounds, then they can add that themselves and see the prediction

Reply:
The issue is you wouldn't expect the user to copy and paste stats from a different website into this. It would be easier for them to input the teams themselves. Also if we allow users to modify data like that doesn't that compromise the ml algorithm as the data used wouldn't be accurate, its just the guess of a user?

Reply:
no they choose the teams, but it will show all the stats that are being entered after they enter the team, they then have to option to edit it if they want. For example the consumer might think team 2 will have more rebounds and team 1 will have less steals, so they should be able to edit it if they want to.